docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Package authoring best practices
+description: A general guide of best practices for creating high quality NuGet packages.
+author: chgill-MSFT
+ms.author: chgill
+ms.date: 09/17/20
+ms.topic: conceptual
+---
+
+# Package authoring best practices
```

Comment:
Overall I really like the structure of this doc! Could we also add more visuals as to what we are referring to? I think this would especially help users know "Why" it's important.

Reply:
Not sure if this is what you meant @JonDouglas, but I wanted to +1 the "Why" part. Could we give a short motivation behind **ALL** the best practices? We have them in many cases but not all.

Reply:
Definitely plan to add more visual examples. Also agree that the "Why" need to be more upfront and clear.

Reply:
Can we have a sample package with all the best practices there? http://github.com/nuget/samples 

Then it may be useful to refer to the actual line for reference as part of these metadata suggestions as required.